completions/bash/kpod

```diff
@@ -653,6 +653,17 @@ __kpod_images() {
 	__kpod_q images $images_args | awk "$awk_script" | grep -v '<none>$'
 }
 
+_kpod_attach() {
```

Comment:
You need to add attach below also in completions.

Reply:
fixed